src/App.js

```diff
@@ -2,12 +2,34 @@ import "./App.css";
 import logo from "./logo.png";
 import { Homepage } from "./pages/Homepage/homepage";
 import { Landingpage } from "./pages/Landingpage/landingpage";
+import { Login } from "./pages/Login-Signup/login";
+import { Signup } from "./pages/Login-Signup/signup";
+import { Route, Routes } from "react-router-dom";
+import { Like } from "./pages/Liked/like";
+import { History } from "./pages/History/history";
+import { WatchLater } from "./pages/WatchLater/watchLater";
```

Comment:
Can use barrel exports to develop the component related to importing multiple pages in a single line by making an index.js file in pages folder that exports those components from there itself.

Reply:
Okay, will do it!